### ISSUE TITLE
Modification of README.md file and Contributing.md file.

### ISSUE THREAD

**Issue by rajdeepchakraborty-rc** [subhadipbhowmik/bio-branch#105]
### Is your proposal related to a problem?

As we all have heard the saying, "First Impression is the Last Impression". As a Contributor, it is difficult to contribute to repositories that have **good structure** and **organization**. I want to help organize your repository and make it more readable and accessible to the Contributors. 

### 1) Add animated line separators
Add animated line separators in between sections and also add the GSSoC 2024 Extended accepted image for Contributors to know that this repository is officially accepted for GSSoC contribution. You have added a Icon, but that is quite small and also in a corner. I think replacing it with this banner along with the animated lines will make it not only look more attractive but also increase the readability of the repository.

![image](https://github.com/user-attachments/assets/4dd78a96-9066-4614-bba6-58dd654c3bf7)

### 2) Add a **Team section**
The Team section will look more organized and readable and let the Contributors know whom to contact if they face any issues..

![image](https://github.com/user-attachments/assets/fae867de-c89c-441c-82ba-4b0d6374ef39)

### 3) Modify the Contributors section:
The contributor's section in your README.md can be modified, and I want to try it out so that It updates and shows the contributors as they contribute to the project.
![image](https://github.com/user-attachments/assets/d28ae333-2507-4dbc-b783-1f746c7920d3)

### 1) Modify and simplify the [👨‍💻 Getting Started with Contributions 👩‍💻](https://github.com/subhadipbhowmik/bio-branch/blob/1d06502e0a967e89995fec1dc61e7fc3ad503034/README.md?plain=1#L38) from README.md file and keep it in the CONTRIBUTING.md file in a detailed manner 

Modify the dedicated file for contributors to refer to and understand where to begin with. Merging the README.md with the Contributing section reflects poor documentation style. I will provide a brief idea about the contribution in the **README.md** file and provide a redirect to the newly made **Contributing.md** file for contributors to refer to.
I will also add a redirect to the **Contributing.md** file from the **README.md** file.

### 2) Add Redirect links under the [Need Help? 🤔](https://github.com/subhadipbhowmik/bio-branch/blob/1d06502e0a967e89995fec1dc61e7fc3ad503034/CONTRIBUTING.md?plain=1#L85) Section for new Contributors to refer to incase they need help with any resources:

- Before

The Contributors would find themselves confused with this.
![image](https://github.com/user-attachments/assets/7404deec-4a63-4cf4-a569-00fa0f8a8bfa)

- After

I have added redirect links to GitHub's official pages for the following help. Contributors can head over there and refer for help now.
![image](https://github.com/user-attachments/assets/4386f59f-02db-4db4-959d-9dda40a91133)

### Describe alternatives you've considered

None.

### Additional context

None.

### Requirement/Document

None.

I would like to work on the issues so that I can contribute to making the repository more accessible and friendly for other Contributors.
Please assign this issue to me under level3, gssoc-ext and hacktoberfest-accepted(if available) labels.

**Comment by rajdeepchakraborty-rc** [comment]
@subhadipbhowmik I just wanted to know if this issue should not have a **level3** tag since I am updating a lot of features in the README.md, making it more attractive and readable. I am also making the dedicated Contributing,md file for the contributors, which results in maintaining good documentation practice in your project.

**Comment by rajdeepchakraborty-rc** [comment]
@subhadipbhowmik Could you please share your LinkedIn so that I can make the Team section.

**Comment by rajdeepchakraborty-rc** [comment]
> @subhadipbhowmik Could you please share your LinkedIn so that I can make the Team section.

Nevermind, I got it from your GitHub profile.

**Comment by rajdeepchakraborty-rc** [comment]
@subhadipbhowmik While working, I noticed that there were no links provided in the following section:
![image](https://github.com/user-attachments/assets/7404deec-4a63-4cf4-a569-00fa0f8a8bfa)
The Contributors would find themselves confused with this.
![image](https://github.com/user-attachments/assets/4386f59f-02db-4db4-959d-9dda40a91133)
I have added redirect links to GitHub's official pages for the following help. Contributors can head over there and refer for help now.
